platform/util/platform-util/src/main/java/org/codice/ddf/platform/util/XMLUtils.java

```diff
@@ -111,6 +118,56 @@ public static Result transform(Node nodeXml, TransformerProperties transformerPr
         return transform(new DOMSource(nodeXml), transformerProperties, result);
     }
 
+    /**
+     * @param xml The XML whose root namespace you want
+     * @return Root Namespace
+     */
+    public static String getRootNamespace(String xml) {
+
+        String rootNamespace = null;
+
+        if (xml == null) {
+            return null;
+        }
+
+        initializeXMLInputFactory();
+
+        XMLStreamReader xmlStreamReader = null;
+
+        try (StringReader strReader = new StringReader(xml)) {
+            xmlStreamReader = xmlInputFactory.createXMLStreamReader(strReader);
```

Comment:
XMLStreamReader doesn't extend AutoClosable, so I can't put it in the try-with-resources.